CHANGELOG.md

```diff
@@ -7,11 +7,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Bug fixes: any changes that fix issues or bug in the existing code
 - Dependencies: updates of dependency versions
 
-## [Unreleased] - yyyy-mm-dd
+## [Unreleased] - 2024-01-02
 
 ### Breaking changes
 
 ### New features & improvements
+- Adds `bodyDoesNotContain()` method to create assertions checking for non-existent values in body content
```

Comment:
We don't use third form here -> `Add` instead of `Adds`

Also, perhaps you can simplify it like this:

- Add `bodyDoesNotContain()` assertion function to check if a response body doesn't contain specific values